README.md

```diff
@@ -1 +1,52 @@
-# BookStore-redux
\ No newline at end of file
+# Describtion
+
+My BookStore .
+
+![screenshot](./src/Assets/)
+
+
+## Installation
+
+- Clone the repository , use  `git@github.com:SamanAtashi/BookStore.git` 
+- open terminal and change Directory, use `cd BookStore`
+- Install all Dependencies, use `npm install` or `npm i`
+
+## Usage
+
+### Development server
+- to run the Webapp you need to use `npm run start`
+
+You can now view the development server at `localhost:3000`.
+
+
+## Built With
+
+- React.js
+- Redux
+- bookstore API
+- redux-thunk middleware
+
+
+## Author
+
+👤 **Saman Atashi**
+
+- Twitter: [@AtashiSaman](https://twitter.com/AtashiSaman)
+- GitHub: [@SamanAtashi](https://github.com/SamanAtashi)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/saman-atashi/)
```

Comment:
- [ ] Also, this link appears broken. It will be nice to fix this as it makes your repo look unprofessional.